soql-analyzer/src/main/scala/com/socrata/soql/SoQLAnalyzer.scala

```diff
@@ -349,20 +349,20 @@ class SoQLAnalyzer[Type](typeInfo: TypeInfo[Type],
 
   def analyzeInOuterSelectionContext(initialCtx: AnalysisContext)(lastQuery: Analysis, query: Select): Analysis = {
     val prevCtx = contextFromAnalysis(lastQuery)
-    val nextCtx = prevCtx ++ initialCtx.filterKeys(qualifier => TableName.PrimaryTable.qualifier != qualifier)
+    val nextCtx = prevCtx ++ initialCtx.view.filterKeys(qualifier => TableName.PrimaryTable.qualifier != qualifier)
```

Comment:
In 2.12 and previous, `filterKeys` was a lazy operation that implicitly returned a view on the original map rather than a copy of the map with the filtered keys removed.  In 2.13 it was made explicit (and I added the `filteredKey` method to `IterableView`-of-`Map` in the `com.socrata.soql.collection` package object; ditto `mapValues` which also created an implicit view before)